scripts/deploy-docker.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+echo Deploying contracts to anvil...
+
+mkdir anvil-data
+
+while ! grep -q "Listening on 0.0.0.0:8545" <(anvil --dump-state ./anvil-state.json --host 0.0.0.0); do
+  sleep 1
+done
+
+forge script -f localhost:8545 --broadcast --slow --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 script/LocalTestnet.sol:Deploy
+if [ $? -ne 0 ]; then
+  exit 1
```

Comment:
Can we replace the manual error handling with the classic bash spell `set -euxo pipefail` at the top of the script, so it bubbles automatically?

Reply:
Done 👍